### ISSUE TITLE
  Error detected while processing function pandoc#command#Pandoc[2]..provider#python3#Call

### ISSUE THREAD

**Issue by gleachkr** [vim-pandoc/vim-pandoc#465]
This is rather similar to #360, but basically on the first invocation of `Pandoc`, I get:

```
Error detected while processing function pandoc#command#Pandoc[2]..provider#python3#Call:                                    
line    1:                                                                                                                   
E5108: Error executing lua Vim:/home/graham/.local/share/nvim/lazy/vim-pandoc/python3/vim_pandoc/command.py:217: SyntaxWarning: invalid escape sequence '\ ' 
```

Subsequent invocations seem to work fine. Thanks for this great plugin!

**Comment by p3palazzo** [comment]
Same still happening with Neovim v0.12.2 and pynvim v0.6.0, loading vim-pandoc plugin through Lazy.nvim. Fails the first time, succeeds afterwards.

**Comment by switzel** [comment]
I also encountered this. The file contains several strings that have escaped whitespaces "\ " which ought not be escaped because a whitespace is not a special character. I don't know why that is (perhaps some backward compatibility issue?), but removing the escapes would resolve the problem. The reason why it occurs only once is that its the python compiler that sends this *warning* (technically nothing is failing) when comiling the code. Once it is compiled, the compiled version is run and the warning is not raised again.

So in summary one should remove the escapes but one can also ignore the warnings.
